starter-code/scripts/articleView.js

```diff
@@ -77,12 +96,20 @@ articleView.setTeasers = function() {
   //       "Read On" link once it has been clicked. Be sure to prevent the default link-click action!
   //       Ideally, we'd attach this as just 1 event handler on the #articles section, and let it
   //       process any .read-on clicks that happen within child nodes.
-
+  $('.read-on').click(function(event){
```

Comment:
This click event is almost working. If I click on one read-on, all of them expand, not just the one I clicked on. If I click on one read on, only that one should expan!